x/superfluid/spec/README.md

```diff
@@ -6,6 +6,647 @@ parent:
 -->
 
 # Superfluid Staking
+<<<<<<< HEAD
```

Comment:
Merge conflict?

Reply:
Uhh... strange! I thought I had already accepted current version before pushing... Fixing now.

Reply:
Patched & waiting for tests to pass.